### ISSUE TITLE
Could not download torrent file: 404

### ISSUE THREAD

**Issue by bigjimbomack1** [DASPRiD/Torrent-to-Web#13]
I've used this add-on in Chrome and Firefox and LOVE it!

Firefox is my main browser right now, and just within the last couple of weeks I've been unable to send torrents to my ruTorrent web client, getting this error; **"could not download torrent file: 404"**. I've made some modifications to about:config recently, and wondered if one of those changes had caused the problem. I revisited the changes and have toggled changes on/ off to no avail.

I'd really like to keep using this add-on in Firefox. Any help or a "point in the right direction" would be most welcome.

Thanks.

![image](https://user-images.githubusercontent.com/38966588/39609949-6b8040d6-4f3c-11e8-992b-87c2b4db21c5.png)

**Comment by DASPRiD** [comment]
Well, 404 means "file not found", so that suggests that the link you try to add leads to a non-existing file. Without further information I can't be of more help though.

**Comment by bigjimbomack1** [comment]
I'm pretty sure it's something to do with the settings in FF, I just don't know where to look, I'm using the add-on in Chrome, with exactly the same configuration and it works perfectly.

**Comment by DASPRiD** [comment]
That's kinda funny, I never published this addon in the chrome web store, where did you get that from?

Did you try to use the addon in a clean Firefox profile?

**Comment by bigjimbomack1** [comment]
Sorry, my bad. The Chrome add-on is called "Add to Remote WebUI". 

Is there any other information I can give you to get this figured out? By the way, I tried a similar add-on from the Firefox Add-on repository, and it also doesn't work.

**Comment by DASPRiD** [comment]
Well, I remember an old bug (#9), but that was a bug in Firefox which was apparently fixed. A detailed discussion about that was done in #12.

**Comment by DASPRiD** [comment]
TL;DR: It was something about not sending third party cookies.

**Comment by bigjimbomack1** [comment]
Ok. I'll check that. Cookie settings are something I have been tinkering with. Cheers.
